xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -1400,6 +1400,12 @@ bool CRenderManager::GetStats(int &lateframes, double &pts, int &queued, int &di
 
 void CRenderManager::CheckEnableClockSync()
 {
+  if (CSettings::GetInstance().GetBool(CSettings::SETTING_VIDEOPLAYER_USEDISPLAYASCLOCK))
+  {
+    m_clockSync.m_enabled = true;
```

Comment:
you are wrong. it calculates the offset between actual pts and vblank. assume clock is started 20ms after vblank has occurred. a/v sync would be off by 20ms. the logic here calculates this offset but this won't work with a moving target, means if refresh rate does not exactly match video fps.

it is not related to passthrough or rr for PCM.